test/checkpoint.test.js

```diff
@@ -1,28 +1,43 @@
 var async = require('async');
 var loopback = require('../');
-
-// create a unique Checkpoint model
-var Checkpoint = loopback.Checkpoint.extend('TestCheckpoint');
-
-var memory = loopback.createDataSource({
-  connector: loopback.Memory
-});
-Checkpoint.attachTo(memory);
+var expect = require('chai').expect;
 
 describe('Checkpoint', function() {
-  describe('current()', function() {
+  var Checkpoint;
+  describe('bump and current', function() {
+    beforeEach(function() {
+      // create a unique Checkpoint model
+      Checkpoint = loopback.Checkpoint.extend('TestCheckpoint');
+      var memory = loopback.createDataSource({
+        connector: loopback.Memory
+      });
+      Checkpoint.attachTo(memory);
+    });
     it('returns the highest `seq` value', function(done) {
       async.series([
-        Checkpoint.create.bind(Checkpoint),
-        Checkpoint.create.bind(Checkpoint),
+        Checkpoint.bump.bind(Checkpoint),
+        Checkpoint.bump.bind(Checkpoint),
         function(next) {
           Checkpoint.current(function(err, seq) {
-            if (err) next(err);
             expect(seq).to.equal(3);
-            next();
+            next(err);
```

Comment:
I think this may lead to difficult-to-troubleshoot situations. When `Checkpoint.current` fails with an error, your version will report an assertion failure (`expected 3, got 1`) and the error details will be lost. Could you please revert this change? Unless I am missing something...